Source/CarthageKit/Project.swift

```diff
@@ -1107,7 +1140,9 @@ private func platformForFramework(_ frameworkURL: URL) -> SignalProducer<Platfor
 /// Sends the URL to each framework bundle found in the given directory.
 private func frameworksInDirectory(_ directoryURL: URL) -> SignalProducer<URL, CarthageError> {
 	return filesInDirectory(directoryURL, kUTTypeFramework as String)
+		.filter { !$0.pathComponents.contains("__MACOSX") }
```

Comment:
This should have a test too.

This change could be moved out into a separate PR.

Reply:
I believe this is tested indirectly via any test that touches `checkoutResolvedDependencies`